CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](https://semver.org).
 
+## 2.2.0
+
+* Add support for version 2.2.0 of the Zivid SDK.
```

Comment:
Should we add any info on the changes to the ROS driver here, or should we rely on the SDK 2.2 changelog and the updated settings in this README?

Reply:
I do not think we need to add info about the new stuff in 2.2. I was thinking maybe we should add that enum settings are now supported in ROS, but I thought it was covered by this point anyway.

Reply:
Yes, I agree, it's apparent from the list of supported settings that enums are supported.